tests/test_kernel32/test_handle.py

```diff
@@ -120,3 +122,86 @@ def test_process_dies_before_timeout(self):
         self.assertEqual(
             WaitForSingleObject(hProcess, library.INFINITE),
             library.WAIT_OBJECT_0)
+
+
+class TestGetHandleInformation(TestCase):
+    """
+    Tests for :func:`pywincffi.kernel32.GetHandleInformation`
+    """
+    def test_get_handle_info_stdin(self):
+        _, library = dist.load()
+        stdin_handle = GetStdHandle(library.STD_INPUT_HANDLE)
+        handle_flags = GetHandleInformation(stdin_handle)
+        inherit = handle_flags & library.HANDLE_FLAG_INHERIT
+        expected = (0, 1)
```

Comment:
I investigated this:
* PEP 0446 says nothing about STDIN: it only applies to "newly created" files.
* The first commit of the test did just that: it tested for specific values depending on Python version (non-inheritable from 3.4 onwards, inheritable otherwise). It failed under Python 3.4 precisely because because of that, STDIN came out as inheritable in every Python version on AppVeyor.
* Microsoft docs seem to indicate that STDIN is always inheritable even though I could not find an explicit statement of that fact. The reasoning seems to make sense, though: if a child process is created and its STDIN is not redirected, it will use the parent's STDIN which needs to be inheritable.

Having said this, the best I could come up with was confirming that we were getting a valid value out of GetHandleInformation for STDIN.